README.md

```diff
@@ -267,6 +267,42 @@ console.log(RNLocalize.uses24HourClock());
 
 ---
 
+### isAutoDateAndTime()
+
+Tells if the automatic date & time setting is enabled on the phone. **Android only**
+
+#### Method type
+
+```ts
+type isAutoDateAndTime = () => boolean;
```

Comment:
I don't know if it's a good practice, but maybe `type isAutoDateAndTime = () => boolean | undefined;` is better since it will returns `undefined` on iOS.

Reply:
this is something I just went through on react-native-device-info - we have so many platform-specific APIs, and what to return on the non-implementing platforms? I decided to simply decide on a default return value in case the value is not known (because of permissions or something) or is not implemented, and it is returned in all cases we don't positively know. For this then I would simply platform-check in the Javascript before going down to the native module, and for all non-implementing platforms I would return false (as that is my default choice for booleans). I like that more personally than return checking but I know this is a matter of style and I am fine doing it your suggested way, or with default to false. Just wanted to offer the option. Think about it then be explicit about what style you want and I will do exactly that :-)